src/PredicateTest1.java

```diff
@@ -0,0 +1,28 @@
+import java.util.Scanner;
+import java.util.function.Predicate;
+
+public class PredicateTest1 {
+	private static Scanner sc;
+	static {
+		sc = new Scanner(System.in);
+	}
+
+	public static void main(String[] args) {
+		Predicate<Integer> predicate = (age) -> {
+			if (age >= 18) {
```

Comment:
No need of if else condition just return condition , It will take care true and false.